parlai/core/agents.py

```diff
@@ -289,12 +289,13 @@ def load_agent_module(opt):
         # only override opts specified in 'override' dict
         if opt.get('override'):
             for k in opt['override']:
-                v = opt[k]
-                if str(v) != str(str(new_opt.get(k, None))):
-                    print("[ warning: overriding opt['" + str(k) + "'] to " +
-                          str(v) + " (previously:" +
-                          str(str(new_opt.get(k, None))) + ") ]")
-                new_opt[k] = v
+                if k in opt:
```

Comment:
what is this? another override change.. i thought this was all done. you shouldnt hide this within a big PR
@emilydinan

Reply:
ah yeah I had a small error when I tried to load the model and included a key that wasn't there on the commandline, let me see

Reply:
maybe      v = opt.get(k, None)   ?

Reply:
thanks for fixing!